src/src/PluginStructs/P002_data_struct.cpp

```diff
@@ -1197,5 +1184,65 @@ float P002_data_struct::mapADCtoFloat(float float_value,
   return float_value;
 }
 
+void P002_data_struct::setTwoPointCalibration(
+  struct EventStruct *event,
+  float adc1,
+  float adc2,
+  float out1,
+  float out2)
+{
+    P002_CALIBRATION_POINT1 = lround(adc1);
+    P002_CALIBRATION_POINT2 = lround(adc2);
+    P002_CALIBRATION_VALUE1 = mapADCtoFloat(
+      P002_CALIBRATION_POINT1,
+      adc1, adc2,
+      out1, out2);
+    P002_CALIBRATION_VALUE2 = mapADCtoFloat(
+      P002_CALIBRATION_POINT2,
+      adc1, adc2,
+      out1, out2);
+}
+
+
+/*****************************************************
+ * plugin_write
```

Comment:
This comment doesn't match the method.

Reply:
... indeed, later decided to change it into `plugin_set_config`